stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ActivityTracker.java

```diff
@@ -107,10 +107,17 @@ public void remove(Activity activity) {
     }
   }
 
-  public List<Activity> getActivitiesView() {
+  public List<Activity> getActivitiesList() {
```

Comment:
Well.. It returns a list of activities, so I thought it made sense to say that it returns an ActivitiesList (instead of ActivitiesView). In case it was intentionally named that way I can rever that (i'm assuming it was a mistake)